.gitignore

```diff
@@ -0,0 +1,7 @@
+/node_modules
+/package-lock.json
+/package.json
```

Comment:
package json and lock files should not be ignored.
Please read what is the purpose of lock files

Reply:
Thanks, corrected.